manifests/r/RealVNC/VNCViewer/7.0.1/RealVNC.VNCViewer.installer.yaml

```diff
@@ -0,0 +1,31 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: RealVNC.VNCViewer
+PackageVersion: 7.0.1
+Platform:
+- Windows.Desktop
+MinimumOSVersion: 10.0.0.0
+InstallModes:
+- interactive
+- silent
+InstallerSwitches:
+  Silent: /qn REBOOT=ReallySuppress
+  SilentWithProgress: /qn REBOOT=ReallySuppress

```

Comment:
```suggestion
  Silent: /qn
  SilentWithProgress: /qn
```

Reply:
The installer doesn't like the argument. 
![image](https://user-images.githubusercontent.com/114952565/219741276-db3d336e-5f76-45fa-bfd2-9c0073d1d9b3.png)